keyboards/thevankeyboards/minivan/keymaps/king/rules.mk

```diff
@@ -0,0 +1,18 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in 
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
Only the build options that differ from the keyboard-level `rules.mk` need to be specified here. It can simply be:

```make
BOOTMAGIC_ENABLE = no
CONSOLE_ENABLE = no
```